opts/global.go

```diff
@@ -1183,6 +1211,7 @@ type RadiusAxis struct {
 	Inverse       bool      `json:"inverse,omitempty"`
 }
 
+var commentPat = regexp.MustCompile(`(//.*)\n`)
 var funcPat = regexp.MustCompile(`\n|\t`)
```

Comment:
How to indicate whether it is a `// comment` or a customer string contains `//` via this regex seems hard.

Reply:
Are you meaning that `commentPat` should be `doubleSlashCommentPat`?

Or... if a customer string contains `//`, they should use `FuncNoStripOpts` ([link](https://github.com/go-echarts/go-echarts/pull/285/files#diff-c15965a03a4b7f8756def936ca13dace4e9a0a8c3dee2182d916629b4289b1ddR1236), although I see I need to fix the documentation).
I'm happy to switch the default to remain no-strip and have a `FuncStripCommentOpts`, which was my original idea..... the name is clear although we could make the [regex more complicated](https://stackoverflow.com/a/17579177/2479314).
This is almost a whole other issue, but I just realized it would be great if a function even pre-flight checks the JavaScript --  I've wasted because incorrectly using `"string"` vs `'string'` !!  Similarly here, `//` ruins the JavaScript injection..

Reply:
Yep, I think we could make the FuncNoStripOpts as the default and provide some useful Func such as `FuncStripCommentOpts`.  besides, In the future, we could provide new options to let users custom their own `fn` to make function chain to process js func further, if it really needs.